misfitapp/models.py

```diff
@@ -164,13 +192,26 @@ def import_from_misfit(cls, misfit, uid, object_id=None):
 
 @python_2_unicode_compatible
 class Device(MisfitModel):
-    id = models.CharField(max_length=MAX_KEY_LEN, primary_key=True)
-    user = models.ForeignKey(UserModel)
-    device_type = models.CharField(max_length=64)
-    serial_number = models.CharField(max_length=100)
-    firmware_version = models.CharField(max_length=100)
-    battery_level = models.SmallIntegerField()
-    last_sync_time = models.DateTimeField(null=True, blank=True)
+    """
+    A Misfit device:
+    https://build.misfit.com/docs/cloudapi/api_references#device
+    """
+    id = models.CharField(
+        max_length=MAX_KEY_LEN,
+        primary_key=True,
+        help_text='The device ID assigned by Misfit')
+    user = models.ForeignKey(UserModel, help_text="The device's user")
+    device_type = models.CharField(
+        max_length=64,
+        help_text='The device type. At this time, will always be "shine"')
```

Comment:
@brad I know Misfit's docs say this, but there are some other values that Misfit will send besides `shine`. #9